src/app/app-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { PostComponent } from './posts/post/post.component';
+import { PostsComponent } from './posts/posts.component';
+
+
+
+const routes: Routes = [
+  { path: '', redirectTo: '/posts', pathMatch: 'full' },
+  { path: 'posts', component: PostsComponent },
```

Comment:
All the routes for posts should be in PostsModule : src/app/posts/posts.routing.module.ts